packages/4-xdapp-layer/aptos/tsconfig.json

```diff
@@ -1,17 +1,10 @@
 {
+  "extends":"../../../tsconfig.cjs.json",
```

Comment:
Should be able to use this to tweak compile settings across the board for builds in the main `tsconfig.cjs.json` file.

Note I _still_ need to provide any relative paths in this instance since otherwise they'll be interpreted incorrectly